README.md

```diff
@@ -155,6 +155,18 @@ print(window:get_word() == nil)
 
 - Nothing. Just resets the window.
 
+#### __tostring
+```lua
+local win = sax.window.new(4, 2, 4)
+for i=1,4 do win:add(1.5) end
+print(win)
+-- prints CC
+```
+
+*Return*  
+
+- string representing this window in SAX notation, throws error if window is not complete
```

Comment:
From a usability perspective throwing an error in the incomplete case is a bit of a pain. Technically the print would have to be wrapped in a pcall and it is inconsistent with get_word which returns a nil if it is incomplete